catkin_tools/context.py

```diff
@@ -194,6 +196,8 @@ def __init__(
         :type cmake_args: list
         :param make_args: extra make arguments to be passed to make for each package
         :type make_args: list
+        :param internal_make_jobserver: true if this configuration should use an internal make jobserv
+        :type internal_make_jobserver: bool
```

Comment:
Again, throughout the code, `use_internal_make_jobsever` or something similar that indicates this is an option and not the actually job sever would be better.